fortios/resource_system_setting_dns.go

```diff
@@ -26,6 +26,13 @@ func resourceSystemSettingDNS() *schema.Resource {
 				Optional: true,
 				Default:  "208.91.112.52",
 			},
+			"domain": &schema.Schema{
```

Comment:
You should be raising a PR against fgtdev/fortios-sdk-go, get that merged and tagged, then vendor the changes to here for this PR.

Reply:
@davewongillies  have created the same https://github.com/fgtdev/fortios-sdk-go/pull/2